test/integration/targets/uri/tasks/install-proxy-and-test.yml

```diff
@@ -0,0 +1,40 @@
+# tinyproxy not available in RHEL repos, and no easy install method for us on macOS
```

Comment:
Would `proxy.py` cover the testing needs? It'd be installable on a non-system level in any virtualenv.

Reply:
I've made this change, although not pushed yet.  I've split it into it's own `setup_proxy` target.